src/pages/Search/SearchBooks.js

```diff
@@ -0,0 +1,139 @@
+  { filter: '제목', placeholder: '제목을 입력하세요' },
+  { filter: '출판사', placeholder: '출판사를 입력하세요' },
+];
+
+export default function SearchBooks() {
+  const [filter, setFilter] = useState('전체');
+  const [visible, setVisible] = useState(false);
+  const [placeholder, setPlaceholder] = useState('검색어를 입력하세요.');
+  const [searchValue, setSerchValue] = useState('');
+
+  const handleFilter = (filter, placeholder) => {
+    setFilter(filter);
+    setPlaceholder(placeholder);
+    setVisible(false);
+  };
+
+  const onChangeSearchValue = e => {
+    setSerchValue(e.target.value);
+  };
+
+  const handleCleanSearch = e => {
+    setSerchValue('');
+  };
+
+  return (
+    <div>
+      <Filtering>
+        <FilterBtn
+          onClick={() => {
+            setVisible(!visible);
+          }}
+        >
+          {filter}
+          <IconBox>
+            <i className="fas fa-angle-down" />
+          </IconBox>
+        </FilterBtn>
+        {visible && (
+          <FilterWrapper>
+            {filters.map(({ filter, placeholder }) => {
+              return (
+                <Filters
+                  key={filter}
+                  onClick={() => handleFilter(filter, placeholder)}
+                >
+                  {filter}
+                </Filters>
+              );
+            })}
+          </FilterWrapper>
+        )}
+      </Filtering>
+      <SearchBar>
+        <IconBox iconSize="1.2rem">
+          <i className="fas fa-search" />
+        </IconBox>
+        <SearchInput
+          placeholder={placeholder}
+          onChange={onChangeSearchValue}
+          value={searchValue}
+        />
+        <IconBox iconSize="0.8rem" onClick={handleCleanSearch}>
+          <i className="fas fa-times-circle" />
+        </IconBox>
+      </SearchBar>
+    </div>
+  );
+}
+
+const Filtering = styled.div`
+  position: relative;
+`;
+
+const FilterBtn = styled.button`
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  width: 5rem;
+  height: 2rem;
+  margin: 15px 0px;
+  cursor: pointer;
+  border: 2px solid ${props => props.theme.grey};
+  border-radius: 10px;
+  color: ${props => props.theme.deepGrey};
+  font-weight: 600;
+`;
+
+const FilterWrapper = styled.span`
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 0;
+  width: 5rem;
+  padding: 5px;
+  border: 1px solid ${props => props.theme.grey};
+  box-shadow: 1px -1px 10px ${props => props.theme.grey};
+  border-radius: 10px;
+  background-color: white;
+`;
+
+const Filters = styled.li`
+  display: inline;
+  cursor: pointer;
+  font-size: 0.8rem;
+  font-family: 'Noto Sans KR', sans-serif;
+  color: ${props => props.theme.deepGrey};
+`;
+
+const SearchBar = styled.div`
+  position: relative;
+  padding: 15px 10px;
+  border-bottom: 2px solid ${props => props.theme.black};
+  color: ${props => props.theme.deepGrey};
+`;
+
+const SearchInput = styled.input`
+  width: 93%;
+  margin-left: 10px;
+  color: ${props => props.theme.black};
```

Comment:
```suggestion
  color: ${({ theme }) => theme.black};
```
함수 인자 자리를 바로 구조분해할당 해줄 수도 있습니다

Reply:
수정 완료했습니다